Mathlib/Data/Set/Ncard.lean

```diff
@@ -117,13 +125,13 @@ theorem ncard_ne_zero_of_mem (h : a ∈ s) (hs : s.Finite := by toFinite_tac) :
   ((ncard_pos hs).mpr ⟨a, h⟩).ne.symm
 #align set.ncard_ne_zero_of_mem Set.ncard_ne_zero_of_mem
 
-theorem finite_of_ncard_ne_zero (hs : s.ncard ≠ 0) : s.Finite :=
+theorem Finite_of_ncard_ne_zero (hs : s.ncard ≠ 0) : s.Finite :=
```

Comment:
Here and elsewhere: these changes to capitalize `finite` to `Finite` are a misunderstanding of the naming conventions. It is `Set.Finite` with a capital `F` because it is is a `Prop`, but in declarations which are *proofs* of a proposition, references to `Set.Finite` are decapitalized (and often the `Set` part is omitted) to `finite`.
```suggestion
theorem finite_of_ncard_ne_zero (hs : s.ncard ≠ 0) : s.Finite :=
```